spec/query_spec.cr

```diff
@@ -57,4 +57,34 @@ describe Query do
       SQL
     end
   end
+
+  describe "#[]" do
+    context "with one argument" do
+      query = Query(User)[42]
+
+      it "generates valid query" do
+        query.to_s.should eq <<-SQL
+        SELECT * FROM users WHERE (id = ?)
```

Comment:
Any advantage to a `LIMIT 1` here?

Reply:
@jackturnbull I've googled it before and the answer is *nope*